sdcm/cluster.py

```diff
@@ -2566,6 +2566,83 @@ def kill_stress_thread_bench(self):
                 if kill_result.exit_status != 0:
                     self.log.error('Terminate scylla-bench on node %s:\n%s', loader, kill_result)
 
+    def run_fullscan(self, loader_node, db_node):
+        """Run cql select count(*) request
+
+        Get all non-system keyspaces
+        Get all tables from randomly choosen ks
+        Request selecl count(*) for ks and randomly choosen cf
+
+        Arguments:
+            loader_node {BaseNode} -- loader cluster node
+            db_node {BaseNode} -- db cluster node
+
+        Returns:
+            object -- object with result of remoter.run command
+        """
+        cmd = 'describe keyspaces'
+        result = loader_node.remoter.run('cqlsh --execute "{}" {}'.format(cmd, db_node.private_ip_address), verbose=False)
+        ks = random.choice([ks.strip() for ks in result.stdout.split() if not ks.startswith('system')])
+
+        cmd = 'use {}; describe tables'.format(ks)
+        result = loader_node.remoter.run('cqlsh --execute "{}" {}'.format(cmd, db_node.private_ip_address), verbose=False)
+        cf = random.choice(result.stdout.split())
+
+        self.log.info('Fullscan for keyspace {} cf {}'.format(ks, cf))
+        cmd = 'select count(*) from {}.{}'.format(ks, cf)
+        result = loader_node.remoter.run('cqlsh --execute "{}" {}'.format(cmd, db_node.private_ip_address), verbose=False)
+
+        return result
+
+    def run_fullscan_thread(self, db_nodes, timeout=60, duration=60):
+        """Run in thread request from loader node
+
+        Run on randomly choosen loader node the cql request to
+        randomly choosen db node during time duration in seconds and with
+        interval timeout in seconds
+
+        Arguments:
+            db_nodes {list} -- list of db nodes db_cluster.nodes
+
+        Keyword Arguments:
+            timeout {int} -- time interval (default: {60})
+            duration {int} -- duration of running (default: {60})
+        """
+
+        def run_in_thread():
+            """Running in thread cql request
+
+            Get all non-system keyspaces
+            Get all tables from randomly choosen ks
+            Request selecl count(*) for ks and randomly choosen cf
+            Repeat until duration with interval timeout
+
+            """
+            st = cr = time.time()
```

Comment:
can you give more meaningful names for `st` and `cr`?

Reply:
consider using `utils.py::log_run_info` decorator, it caclulates and prints runtime

Reply:
changed